core/templates/components/question-directives/question-editor/question-editor.directive.ts

```diff
@@ -63,14 +63,14 @@ angular.module('oppia').directive('questionEditor', [
         'EditabilityService', 'EditableQuestionBackendApiService',
         'QuestionObjectFactory',
         'INTERACTION_SPECS', 'StateEditorService', 'ResponsesService',
-        'SolutionValidityService', 'QuestionUpdateService',
+        'SolutionValidityService', 'QuestionUpdateService', 'LoaderService',
```

Comment:
Please add this in sorted order. (Ditto everywhere)

Reply:
Ah, missed this one, Will change it.

Reply:
Shouldn't be anywhere else though.